mlflow/utils/search_utils.py

```diff
@@ -456,14 +457,22 @@ def _validate_order_by_and_generate_token(cls, order_by):
                 error_code=INVALID_PARAMETER_VALUE,
             )
         statement = parsed[0]
+        ttype_for_timestamp = (
+            TokenType.Name.Builtin
+            if Version(sqlparse.__version__) >= Version("0.4.3")
+            else TokenType.keyword
+        )
+
         if len(statement.tokens) == 1 and isinstance(statement[0], Identifier):
             token_value = statement.tokens[0].value
         elif len(statement.tokens) == 1 and statement.tokens[0].match(
-            ttype=TokenType.Keyword, values=[cls.ORDER_BY_KEY_TIMESTAMP]
+            ttype=ttype_for_timestamp, values=[cls.ORDER_BY_KEY_TIMESTAMP]
```

Comment:
we'd better avoid using `timestamp` as table column name, so that it can be parsed as a normal identifier

Reply:
Yep, but we can't change the existing columns.